ng5/src/app/screens/register/register.component.scss

```diff
@@ -0,0 +1,82 @@
+@import '../../../sass/colors.scss';
+
+.register { 
+  align-items: center; 
+  background-color: $grey;
+  border-top: 3px solid $blue;
+  display: flex;
+  flex-direction: column;
+  margin: 150px auto;
+  max-width: 350px;
+  padding: 10px;
+  width: 100%;
+
+  .logo {
+    height: 40px;
+    margin: 20px 20px 10px;
+  }
+
+  .alert {
+    background-color: $red;
+    border-radius: 10px;
+    color: $white;
+    display: flex;
+    justify-content: center;
+    margin: 0 20px;
+  }
+
+  .subtitle { 
+    font-size: 12px;
+    letter-spacing: 8px;
+  }
+}
+
+.register-form {
+  display: flex;
+  flex-direction: column;
+  margin: 10px;
+  width: 100%;
+  
+  .label-input {
+    display: flex;
+    flex-direction: column;
+    font-size: 15px;
+    margin: 10px;
+  }
+
+  .btn-login {
```

Comment:
add some generic button classes on another file. You can have: .button-primary and .button-secondary.
A good practice is to have a buttons.scss